nvme-print.c

```diff
@@ -169,13 +169,13 @@ static void json_nvme_id_ns(struct nvme_id_ns *ns, bool cap_only)
 		}
 
 		json_object_add_value_int(root, "mssrl", le16_to_cpu(ns->mssrl));
-		json_object_add_value_int(root, "mcl", le32_to_cpu(ns->mcl));
+		json_object_add_value_uint(root, "mcl", le32_to_cpu(ns->mcl));
```

Comment:
We should also update the normal terminal output for this value. The printf format string `%d` is used currently.